src/System.Management.Automation/FormatAndOutput/common/BaseFormattingCommand.cs

```diff
@@ -758,6 +758,12 @@ public SwitchParameter AutoSize
         }
         private Nullable<bool> _autosize = null;
 
+        /// <summary>
+        /// Gets or sets if header is repeated per screen.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter RepeatHeader { get; set; }
```

Comment:
I wonder if this parameter should take an optional integer row count value?  That way a user can control how often the header is repeated.  For accessibility, a visual impaired person may want the header repeated more often.